cogs/rss/__init__.py

```diff
@@ -0,0 +1,34 @@
+"""
+Init file for RSS cog.
+"""
+
+
+import logging
+import os
+
+from redbot.core.bot import Red
+from .rss import RSSFeed
+
+LOG_FOLDER = "log/lui-cogs/rss"
+LOGGER = None
+def checkFolders():
+    """Make sure folder for logs is available."""
+    if not os.path.exists(LOG_FOLDER):
+        os.makedirs(LOG_FOLDER)
+
+def setup(bot: Red):
+    """Add the cog to the bot."""
+    global LOGGER # pylint: disable=global-statement
+    LOGGER = logging.getLogger("red.RSSFeed")
+    if LOGGER.level == 0:
+        # Prevents the LOGGER from being loaded again in case of module reload.
+        LOGGER.setLevel(logging.INFO)
+        handler = logging.FileHandler(filename="data/rss/info.log",
+                                      encoding="utf-8",
+                                      mode="a")
+        handler.setFormatter(logging.Formatter("%(asctime)s %(message)s",
+                                               datefmt="[%d/%m/%Y %H:%M:%S]"))
+        LOGGER.addHandler(handler)
+    rssCog = RSSFeed(bot)
```

Comment:
Please do **one** of the following:
- Modify the RSSFeed constructor to accept a `logging.Logger` object, and assign it to `self.logger` (might be better).
- Add the following line so that entries show up in the logger file you've set up above:
```suggestion
    rssCog = RSSFeed(bot)
    rssCog.logger = LOGGER
```

Reply:
Not quite sure how to set up the constructor properly so I'll leave that for later, For the time being I've performed the second fix.